tests/network/localnet/liblocalnet.py

```diff
@@ -1,15 +1,60 @@
+import tests.network.libs.nodenetworkconfigurationpolicy as libnncp
 from libs.net import netattachdef
-from libs.net.vmspec import add_network_interface, add_volume_disk
+from libs.net.traffic_generator import Client, Server
+from libs.net.vmspec import IP_ADDRESS, add_network_interface, add_volume_disk, lookup_iface_status
 from libs.vm.affinity import new_pod_anti_affinity
 from libs.vm.factory import base_vmspec, fedora_vm
 from libs.vm.spec import CloudInitNoCloud, Interface, Metadata, Multus, Network
 from libs.vm.vm import BaseVirtualMachine, cloudinitdisk_storage
 from tests.network.libs import cloudinit
 
-NETWORK_NAME = "localnet-network"
+_IPERF_SERVER_PORT = 5201
+NNCP_INTERFACE_TYPE_OVS_BRIDGE = "ovs-bridge"
 
 
-def localnet_vm(namespace: str, name: str, network: str, cidr: str) -> BaseVirtualMachine:
+def run_vms(
+    vms: tuple[BaseVirtualMachine, BaseVirtualMachine],
```

Comment:
```suggestion
    vms: tuple[BaseVirtualMachine],
```
can be any number of VMs

Reply:
Or even a more elegant variadic: `*vms: BaseVirtualMachine`.
The return will be then `tuple[BaseVirtualMachine]`.

Reply:
Unfortunately this caused a problem with flake8.
I opened a new thread about it on the new code.